discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
```

Comment:
With Discord's roll-out of 2FA and preference for token-only logins, there is now a good reason for libraries to not expose email/password logins. I don't think we should have a column for supportsUserLogin.